masschallenge/django-accelerator
# [AC-6852] Delete obsolete fields from OfficeHour model

#### Changes introduced in [AC-6852](https://masschallenge.atlassian.net/browse/AC-6852)
- removed obsolete fields and associated dependencies in tests

#### How to test
There are existing instructions off the AC-4820 PRs ([django-accelerator](https://github.com/masschallenge/django-accelerator/pull/221) and [accelerate](https://github.com/masschallenge/accelerate/pull/2267))

Checkout AC-6852 on accelerate and django-accelerator and run `make migrate`

See below, use this script in the django shell. it will output a mentor and finalist user you can use to masquerade and test office hours
```python
import mc.models as mc
program = mc.Program.objects.filter(program_status="active").first()
program.end_date
from datetime import timedelta
one_day = timedelta(1)
from datetime import datetime
from pytz import utc
program.end_date = utc.localize(datetime.now() + one_day)


mentor_pr = program.programrole_set.filter(user_role__name="MENTOR").first()
mentor = mentor_pr.programrolegrant_set.last().person
finalist_pr = program.programrole_set.filter(user_role__name="FINALIST").first()
finalist = finalist_pr.programrolegrant_set.last().person
finalist, mentor
```

With the output, you will have user IDs you can use to mask as a mentor or finalist to test the office hours functionality e.g. booking, creating

#### Note
There's a related [PR](https://github.com/masschallenge/accelerate/pull/2278). Both of these should be merged as close together as possible
